Make subset of python files PEP8-compatible

vitalii-komisarenko opened the pull request:
Making non-test-related python file pep8-compatible.

vitalii-komisarenko committed:
deploy.py: make code PEP8-compatible

Making deploy.py PEP8-compatible as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
package.py: make code PEP8-compatible

Making package.py PEP8-compatible as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
config.py: make code PEP8-compliant

Making config.py PEP8-compliant as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
build.py: make code PEP8-compliant

Making build.py PEP8-compliant as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

arnout commented on tools/commands/deploy.py:
```diff
@@ -37,15 +35,17 @@ def __connect__(self):
             stdin, stdout, stderr = self.client.exec_command(c)
             if stdout.channel.recv_exit_status():
-                logger.error("Command failed:\n{}\nexit code - {}, stderr - {}".format(c, stdout.channel.recv_exit_status(), stderr.read()))
+                logger.error("Command failed:\n{}\nexit code - {}, stderr - {}".format(
```
Maybe we should decide on a coding style for this kind of situation...

```
        "blahblahblah".format(
            x, y, z)
```

or

```
        "blahblahblah".\
            format(x, y, z)
```

Both are a bit ugly, but I have a slight preference for the second option.

Well, actually, I'd really prefer to use f-strings which would remove the need for ever using `"...".format()`, but apparently not everyone has python 3.6 :-(

arnout commented on tools/commands/deploy.py:
```diff
@@ -107,33 +124,39 @@ def mkdir_p(path):
-        modules = deploy_modules if 'all' in args.modules else [m for m in deploy_modules if m in args.modules]
+        if args.verbose:
+            logger.setLevel(logging.DEBUG)
+        modules = deploy_modules if 'all' in args.modules else [
+            m for m in deploy_modules if m in args.modules]
```
Meh, it's a bit silly to fix this since it actually doesn't work any more... There are no longer any modules...

Oh well, let's just fix it for pep8.

vitalii-komisarenko replied:
Fixed for PEP8

arnout commented on tools/commands/config.py:
```diff
@@ -113,15 +123,16 @@ def __guess_setup_id(self):
             from chdlab.commands.config import CHDLAB_config
             return CHDLAB_config().setup_number
-        except:
+        except Exception:
```
The previous commit still leaves a catch-all except, so why not do the same here?

vitalii-komisarenko replied:
flake8 compains with `E722 do not use bare except`, so I add `Exception`

arnout commented on tools/commands/build.py:
```diff
@@ -42,8 +50,11 @@ def make(self):
 
 # cmake-based builder class
+
+
```
The double newline should be *before* the command, the comment is tied to the cmakebuilder class.

arnout commented on tools/commands/build.py:
```diff
@@ -85,8 +96,11 @@ def make(self):
 
 # automake-based builder class
+
+
```
Same here, the comment is tied to the acbuilder class.

arnout commented on tools/commands/build.py:
```diff
@@ -129,6 +143,8 @@ def make(self):
 
 # hostap builder class
+
+
```
And same here again.

arnout commented on tools/commands/build.py:
```diff
@@ -159,12 +175,15 @@ def make(self):
         commands = args.commands
-        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules else [m for m in dep_modules if m in args.modules]
+        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules else [
+            m for m in dep_modules if m in args.modules]
```
I don't think this is a nice way to split. Better split on the else:

```suggestion
        _dep_modules = dep_modules if 'all' in args.modules or 'dep' in args.modules 
            else [m for m in dep_modules if m in args.modules]
```

arnout reviewed (approved):
I think it would be good to merge this soon. There's no need to have everything PEP8 compliant before we merge this.

Also, it may be a good idea to already add the pep8 check to CI, but just allow it to fail.

adam1985d commented:
> Making deploy.py PEP8-compartible as a part of
> preparation for automatic style checks of python files on CI.
> 
> Signed-off-by: Vitalii Komisarenko [vitalii.komisarenko@gmail.com](mailto:vitalii.komisarenko@gmail.com)

I think you meant `compatible` and not `compartible` - both in PR description and in PR name

this comment is also relevant to all commit headlines and messages

vitalii-komisarenko committed:
beerocks_analyzer_widget: make code more PEP8-compliant

Making beerocks_analyzer_widget more PEP8-compliant as
a preparation for automatic style checks of python files on CI

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
beerocks_analyzer.py: make code PEP8-compliant

Making beerocks_analyzer.py PEP8-compartible as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
logger_widget.py: make code PEP8-compliant

Make logger_widget.py PEP8-compliant as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
beerocks_analyzer: make conf1.py PEP8-compliant

Making beerocks_analyzer/conf/conf1.py PEP8-compartible as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
beerocks_message_endian_converter.py: make code more PEP8-compliant

Making beerocks_message_endian_converter.py more PEP8-compartible
as a part of preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko commented:
beerocks_message_endian_converter.py is a python2 file.
Shall it be converted to python3?

tomereli commented:
> beerocks_message_endian_converter.py is a python2 file.
> Shall it be converted to python3?

I think we can delete it. Right @vitalybu ?

rmelotte commented on tools/beerocks_analyzer/beerocks_analyzer.py:
```diff
@@ -13,46 +13,54 @@
+    logger.info("    -ssh_port=[ssh port of GW]                            "
+                "- SSH port used to connect to GW")
+    logger.info("    -docker_container=[name of container]                 "
+                "- Name of the docker container on which to run beerocks_cli")
```
What a mess... This shouldn't even be here, it should be handled by a library like argparse.

For now if you have the opportunity, could you just remove all the extra spaces before the second dash? So the first one would be like this for example:
```
"    -f=<input log file> - input log file"
```
It should make it look a tiny bit less messy.

rmelotte commented on tools/beerocks_analyzer/beerocks_analyzer_widget.py:
```diff
@@ -184,19 +195,25 @@ def getCommandLineArgs(self, argv):
-        self.logger.info("    -print_labels                                         - print log labels to console\n")
-        self.logger.info("    -output_csv                                           - output params to csv file\n")
+        self.logger.info(
+            "    -f=<log file>                                     - input log file\n")
```
Same here, if you have the time/opportunity, please remove all the extra spaces after the single dash.

rmelotte reviewed (approved):
Pretty hard to review in details of course, but overall LGTM.

Can you also add the ones that passes flake8 to `FLAKE8_FILES` in `.gitlab-ci.yml`?

vitalybu commented:
> > beerocks_message_endian_converter.py is a python2 file.
> > Shall it be converted to python3?
> 
> I think we can delete it. Right @vitalybu ?

Yup

vitalii-komisarenko committed:
bcl: removed beerocks_message_endian_converter.py

Removed no longer needed beerocks_message_endian_converter.py

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

vitalii-komisarenko committed:
PEP8: git CI: add list of *.py file to check style

Test code style for python files not involved in testing
Files involved in testing are skipped due to being refactored independently

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

arnout commented on .gitlab-ci.yml:
```diff
@@ -28,7 +28,7 @@ flake8:
   image: pipelinecomponents/flake8:f087c4c
   variables:
-    FLAKE8_FILES: "tests/*.py"
+    FLAKE8_FILES: "tests/*.py tools/beerocks_analyzer/conf/conf1.py tools/beerocks_analyzer/*.py tools/commends/*.py"
```
```suggestion
    FLAKE8_FILES: "tests/*.py tools/beerocks_analyzer/conf/conf1.py tools/commands/*.py"
```

vitalii-komisarenko replied:
beerocks_analyzer already passes the code check.
typo commEnds->commAnds is addressed in https://github.com/prplfoundation/prplMesh/pull/1118

vitalii-komisarenko committed:
PEP8: make connectivity_map_widget.py PEP8-compliant

Making connectivity_map_widget.py PEP8-compatible as a part of
preparation for automatic style checks of python files on CI.

Signed-off-by: Vitalii Komisarenko <vitalii.komisarenko@gmail.com>

Commit:
Merge branch 'master' into task/python-style-check
